### ISSUE TITLE
Mutual SSL Authentication

### ISSUE THREAD

**Issue by araoko** [icza/gowut#48]
Is there any plans to implement mutual authentication such that the gowut server can be configures to request or require client certificate?

**Comment by icza** [comment]
I haven't thought about it. Does anyone else need it?

**Comment by simplepearls** [comment]
Yes, SSL certifiicate authentication is required. And I want to build a product using Gowut, will this product be upgraded to the latest Golang versions?

**Comment by icza** [comment]
There are no planned work on Gowut.
